stories/atoms/Badges.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { StatusBadge } from '@textkernel/oneui';
+
+const meta: Meta<typeof StatusBadge> = {
+    title: 'Atoms/Badges',
+    component: StatusBadge,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof StatusBadge>;
+
+export const _StatusBadge: Story = {
```

Comment:
Since adding an icon though Storybook is not that easy, maybe it is good to have an additional story with the icon inplace.